lib/health_cards/keys.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'openssl'
+require 'jose'
+
+# Methods to generate signing/encryption keys
+module Keys
+  def generate_signing_key(path = nil)
+    key = JOSE::JWS.generate_key({ 'alg' => 'ES256' })
+    save_key(key.to_key, path)
+    key_to_jwk(key, 'sig')
+  end
+
+  def generate_encryption_key(path = nil)
```

Comment:
Hmm this used to be in the spec but it may have been removed as part of the simplification PR. I still see the encryption key in their `.well-known` endpoint so I'll leave it in for now but can be easily removed in the future.

Reply:
Sounds good, I saw the updated version without the encryption key.